src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,54 @@
+package duke.parser;
+
+
+import duke.command.*;
+import duke.tasks.TaskList;
+
+public class Parser {
+    /**
+     * Commands
+     */
+    public static final String LIST = "list";
+    public static final String DONE = "done";
+    public static final String TODO = "todo";
+    public static final String DEADLINE = "deadline";
+    public static final String EVENT = "event";
+    public static final String DELETE = "delete";
+
+    public static final String SPACE_DELIMITER = "\\s+";
+    public static final String START_LINE_DELIMITER = "^";
+
+    public Parser() {
+    }
+
+    public static Command parsedCommand(TaskList taskList, String input) {
+
+        final String[] splitLine = input.split(" ", 2);
+        final String command = splitLine[0];
```

Comment:
Avoid the usage of magic literals (0 and 2 here). Consider declaring final variables and using them instead.